src/project-css/app/Helpers/CMSHelper.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @author Ajay Krishna Teja Kavur
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Helpers;
+
+use App\Models\CMSRecords;
+use App\Models\Collection;
+use App\Helpers\CSVHelper;
+use App\Helpers\TableHelper;
+
+class CMSHelper {
+    /**
+     * CMS Helper
+     *
+     * These are various functions that help with processing
+     * cms files.
+     *
+     */
+
+
+     /**
+      * Check the Record Types table for a matching header for the table
+      *
+      * @param integer $collctnID Contains the collection ID
+      * @param string $recordType Contains the Record type 1A, 1B, etc.
+      * @param integer $fieldCount Contains number of fields expected for the Record
+      *
+      * @author Tracy A. McCormick <tam0013@mail.wvu.edu>
+      * @return unserialized Array if found or null
+      */
+     public function getCMSFields($collctnId, $recordType, $fieldCount) {
+       if (CMSRecords::isCMSRecord($recordType)) {
```

Comment:
This could be a guard clause, however you are passing in RecordType you should already have validated that information before calling this function unless it is direct from the user.